Tests/StencilSwiftKitTests/StringFiltersTests.swift

```diff
@@ -133,6 +133,77 @@ class StringFiltersTests: XCTestCase {
     }
   }
 
+  func testCamelToSnakeCase_WithNoArgsDefaultsToTrue() {
```

Comment:
This test covers the ```?? true``` flow in the line:
```let toLower = try parseBool(from: arguments, index: 0, required: false) ?? true``` because there's no other test for the no arguments case. The other two camelToSnakeCase tests pass arguments.